eng/.docsettings.yml

```diff
@@ -9,8 +9,14 @@ omitted_paths:
   - samples/*
   - common/SmokeTests/*
   - sdk/*/samples/*
+  - sdk/*/*/samples/*
   - sdk/*/swagger/*
   - doc/ApiDocGeneration/*
+  - doc/*
```

Comment:
What is the difference between `doc/*` and `doc/` now?

Reply:
Good point. I'm not certain how I didn't see something directly above it.